cli/src/cluster_query.rs

```diff
@@ -128,6 +135,11 @@ pub fn parse_catchup(matches: &ArgMatches<'_>) -> Result<CliCommandInfo, CliErro
 }
 
 pub fn parse_cluster_ping(matches: &ArgMatches<'_>) -> Result<CliCommandInfo, CliError> {
+    let lamports = if matches.is_present("lamports") {
+        value_t_or_exit!(matches, "lamports", u64)
+    } else {
+        1
+    };
```

Comment:
Adding `.default_value("1")` allows this to be simplified down to `let lamports = value_t_or_exit!(matches, "lamports", u64);` too

Reply:
same as https://github.com/solana-labs/solana/pull/7029#discussion_r347735823!